editor/action_map_editor.cpp

```diff
@@ -682,8 +665,8 @@ InputEventConfigurationDialog::InputEventConfigurationDialog() {
 	// Physical Key Checkbox
 
 	physical_key_checkbox = memnew(CheckBox);
-	physical_key_checkbox->set_text(TTR("Use Physical Keycode"));
-	physical_key_checkbox->set_tooltip(TTR("Stores the physical position of the key on the keyboard rather than the keys value. Used for compatibility with non-latin layouts."));
+	physical_key_checkbox->set_text(TTR("Use Physical Keycode (US QWERTY layout)"));
```

Comment:
```suggestion
	physical_key_checkbox->set_text(TTR("Use Physical Keycode (US QWERTY Layout)"));
```
This uses "Title Case".

Reply:
Sorry. I missed it. Done.